sway/ipc-json.c

```diff
@@ -503,6 +504,28 @@ json_object *ipc_json_describe_input(struct sway_input_device *device) {
 	json_object_object_add(object, "type",
 		json_object_new_string(describe_device_type(device)));
 
+	if (device->wlr_device->type == WLR_INPUT_DEVICE_KEYBOARD) {
+		struct wlr_keyboard *keyboard = device->wlr_device->keyboard;
+		struct xkb_keymap *keymap = keyboard->keymap;
+		struct xkb_state *state = keyboard->xkb_state;
+		xkb_layout_index_t num_layouts = xkb_keymap_num_layouts(keymap);
+		for (xkb_layout_index_t layout_idx = 0;
```

Comment:
Oh, I missed the fact that including `xkb_layout_index_t` here makes the line go over the 80 char limit. Please either declare it right before, or indent with two tabs (instead of aligning).